.changeset/breezy-pianos-slide.md

```diff
@@ -0,0 +1,52 @@
+---
+'@astrojs/cloudflare': minor
+---
+
+Adds the option to set a custom workerEntryPoint. This feature is only supported for Cloudflare Workers and when you use `astro build`. Below is an example on how to use it to register a Durable Object and a queue handler.
+
+```ts
+// astro.config.ts
+import cloudflare from '@astrojs/cloudflare';
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+	adapter: cloudflare({
+		workerEntryPoint: {
+			path: 'src/worker.ts',
+			exports: ['default','MyDurableObject']
```

Comment:
```suggestion
			namedExports: ['MyDurableObject']
```
Just noting that in the docs PR, we're discussing a suggestion like maybe this one. If so, whatever we choose there would need updating here.